test/support/number_validation.rb

```diff
@@ -78,16 +78,5 @@ def test_multiple_of
       # other types are disregarded
       assert_valid schema, {'a' => 'hi'}
     end
-
-    def test_multiple_of_zero
```

Comment:
The schema for this test was actually invalid (`multipleOf` must be greater than 0)